### PR TITLE
Doc updates2

### PR DESCRIPTION
**Pull request by james-s-willis**
Update to `kotekan` compilation instructions.

### CODE REVIEW

File: docs/sphinx/compiling/general.rst [hunk 1]
```diff
@@ -33,16 +33,99 @@ Required for some options
 
 * `libevent <http://libevent.org/>`_
```
Comment (andrerenard) [inline docs/sphinx/compiling/general.rst]: libevent should be moved to the required section.

File: docs/sphinx/compiling/general.rst [hunk 2]
```diff
@@ -33,16 +33,99 @@ Required for some options
...
+* `LAPACK <http://www.netlib.org/lapack/>`_ (Should automatically be installed with OpenBLAS if a Fortran compiler is present on your system)
+* `Blaze <https://bitbucket.org/blaze-lib/blaze/src/master/>`_
+  * Download and unpack into `/opt`
 * OpenSSL (Package is called ``libssl-dev`` on Ubuntu).
-* `pytest-cpp <https://github.com/pytest-dev/pytest-cpp>`_
+* For python packages to be installed for all::
+
+    sudo su
+    cd ~
+    pip3 install python-module
```
Comment (andrerenard) [inline docs/sphinx/compiling/general.rst]: This seems a little out of place here.  Maybe make this a footnote and reference it when suggesting `pip` commands below?

**Comment by andrerenard** [comment]
I think you might need to rescan the repo to get the `branch` build to run.

**Comment by james-s-willis** [comment]
The branch build won't run in the current settings. I will re-enable branch builds until we discuss it at the meeting.

**Pull request merged**